src/queue.c

```diff
@@ -976,6 +976,7 @@ _dispatch_get_mach_host_port(void)
 #include <unistd.h>
 #include <sys/syscall.h>
 
+#ifndef __ANDROID__
```

Comment:
perfect, I just wanted to make sure. we don't care about the performance because this is cached once per thread.